src/amqp_direct_connection.erl

```diff
@@ -38,7 +38,7 @@
                 connected_at
                }).
 
--define(INFO_KEYS, [type]).
+-define(INFO_KEYS, [type, internal_user]).
```

Comment:
Should `internal_user` be exposed to client users via `amqp_gen_connection:info_keys/1`?